react/week1/todo-list/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Over all nice job done. 

Your can remove default styles for the App!

Reply:
done